cli/multisig.go

```diff
@@ -214,6 +214,10 @@ var msigInspectCmd = &cli.Command{
 
 		store := adt.WrapStore(ctx, cbor.NewCborStore(blockstore.NewAPIBlockstore(api)))
 
+		if cctx.NArg() != 1 {
+			return IncorrectNumArgs(cctx)
+		}
+
```

Comment:
this is being done too late, do it at the top like the rest, but also note that it's already there with a `cctx.Args().Present()` check. You could replace it with what you have (just moved up) and preserve the message by using `IncorrectNumArgsWithHint`